content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst

```diff
@@ -148,47 +149,56 @@ specified in the :guilabel:`Work Order Operation` field on the |QCP| form. If a
+taking the measurement, enter it in the :guilabel:`Measure` field of the pop-up window, and then
+click :guilabel:`Validate`.
+
+.. image:: measure_check/measure-check-shop-floor.png
    :align: center
-   :alt: A Measure quality check in the Manufacturing tablet view.
+   :alt: A Measure check in the Shop Floor module.
 
 If the measurement entered is within the range specified in the :guilabel:`Tolerance` section of the
-|QCP|, the quality check passes and the tablet view moves on to the next step of the work order.
-However, if the measurement entered is outside of the specified range, a pop-up window appears,
+|QCP|, the quality check passes and the pop-up window moves on to the next step of the work order.
+However, if the measurement entered is outside of the specified range, a new pop-up window appears,
 titled :guilabel:`Quality Check Failed`.
 
-The body of the :guilabel:`Quality Check Failed` pop-up window shows a warning message that states,
-:guilabel:`You measured # units and it should be between # units and # units.`, as well as the
+The body of the :guilabel:`Quality Check Failed` pop-up window shows a message that states,
+:guilabel:`You measured # units and it should be between # units and # units`, as well as the
 instructions entered in the :guilabel:`Message If Failure` tab of the |QCP|. At the bottom of the
 pop-up, two buttons appear: :guilabel:`Correct Measure` and :guilabel:`Confirm Measure`.
 
-.. image:: measure_check/measure-check-failed.png
+.. image:: measure_check/shop-floor-measure-check-failed.png
    :align: center
-   :alt: The "Quality Check Failed" pop-up window.
+   :alt: The Quality Check Failed pop-up window for a Measure check in the Shop Floor module.
 
 If the measurement was not entered correctly, and should be changed, select :guilabel:`Correct
 Measure`. Doing so opens a new pop-up window, titled :guilabel:`Quality Check`. Enter the corrected
-measurement in the :guilabel:`Measure` field, then click :guilabel:`Validate` to complete the check,
-and move on to the next step of the work order.
+measure in the :guilabel:`Measure` field, and then click :guilabel:`Validate` to complete the check
+and close the pop-up window.
 
 If the measurement was entered correctly, click :guilabel:`Confirm Measure` instead, and the quality
 check fails. Follow any instructions that were listed on the :guilabel:`Quality Check Failed` pop-up
 window.
 
-If a quality alert must be created, do so by clicking the :guilabel:`☰ (three horizontal lines)`
-button, and selecting :guilabel:`Quality Alert` from the :guilabel:`Menu` pop-up window. A
-:guilabel:`Quality Alerts` pop-up window appears, from which a quality alert can be created.
+If a quality alert must be created, exit the pop-up window by clicking the :guilabel:`X` button in
+the top-right corner. Then, click the :guilabel:`⋮ (three vertical dots)` button on the bottom-right
+corner of the work order card to open the :guilabel:`What do you want to do?` pop-up window.
```

Comment:
```suggestion
If a quality alert must be created, exit the pop-up window by clicking the :guilabel:`✖️ (X)`
button in the top-right corner. Then, click the :guilabel:`⋮ (three vertical dots)` button on the
bottom-right corner of the work order card to open the :guilabel:`What do you want to do?` pop-up
window.
```

Reply:
a little separation always makes a doc a bit easier for readers to digest. And if you don't feel like the emoji use is necessary, you don't have to implement it. Just figured it would look a bit nicer. That's up to you. 👍